tests/integration/test_llm_providers_registry.py

```diff
@@ -0,0 +1,34 @@
+from tests.common.request import payload, post_request_localhost
+from tests.common.response import has_success_status
+
+
+def test_llm_providers():
+    reset_result = post_request_localhost(
+        payload("reset_defaults_llm_providers")
+    ).json()
+    assert has_success_status(reset_result)
+
+    response = post_request_localhost(payload("get_providers_and_models")).json()
+    assert has_success_status(response)
+
+    default_providers = response["result"]["data"]
+    first_default_provider = default_providers[0]
+    last_provider_id = default_providers[-1]["id"]
+
+    # Create a new provider
```

Comment:
I didn't wanted to add hardcoded values since they'd be changing with the schema, but you are right. Once we have a stable schema these tests shouldn't change often

I'll create another test (I think this one adds it's own logic so I won't remove it)